pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,32 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
+
+	if follow {
```

Comment:
How is all this in any form implementing a steaming of the logs? AFAICT, this command will just collect the output and return it in the end. Am I missing something here. I don't think that in its current form the we have the right helpers and infrastructure to implement streaming. 

I am also a bit surprised why we are trying to implement this atm. Neither it is part of the sprint afaik, nor is it easy to implement.